apiServer.js

```diff
@@ -129,7 +142,7 @@ app.get('/libraries', function (req, res) {
 
 app.get('/libraries/:library', function (req, res) {
   var results;
-  var fields = (req.query.fields && req.query.fields.split(',')) || false;
+  var fields = safeFields((req.query.fields && req.query.fields.split(',')) || []);
```

Comment:
Fun note about line https://github.com/cdnjs/new-website/blob/0a757fe170b29a241149513293ac19346d877f67/apiServer.js#L159, in JS `[] == true`, so before that would not match while now it might. Do you think that could introduce a bug @MattIPv4? Should we add a test there around `fields.length`?

Reply:
Looking at it, I don't see any issue it'd cause, but yeah, will make that change for consistency.